zipline/finance/slippage.py

```diff
@@ -502,3 +502,30 @@ def get_simulated_impact(self,
     def get_txn_volume(self, data, order):
         volume = data.current(order.asset, 'volume')
         return volume * self.volume_limit
+
+
+class FixedBasisPointsSlippage(SlippageModel):
```

Comment:
We should add tests for this to tests/finance/test_slippage.py.

Reply:
`VolumeShareSlippageTestCase` looks like it would be a reasonable model to look at for adding these.

Reply:
added a test for this slippage model (may change the test if we change the model)